app/classes/Stores/Project.php

```diff
@@ -164,7 +164,6 @@ class Project
             'release' => [
                 'template' => 'apple/release/listing_sept_2015.php',
                 'langfile' => 'apple_description_release.lang',
-                'whatsnew' => 'whatsnew/whatsnew_ios_2.lang',
```

Comment:
If you remove the whatsnew section for Apple, you need to update/comment out the functional tests